configuration-reference/schema.md

```diff
@@ -60,6 +60,29 @@ The Pinot schema is composed of:
 
 The above json configuration is the example of Pinot schema derived from the flight data. As seen in the example, the schema is composed of 4 parts: `schemaName`, `dimensionFieldSpec`, `metricFieldSpec`, and `dateTimeFieldSpec`. Below is a detailed description of each type of field spec.
 
+
+### Data Types
+Data types determine the operations that can be performed on a column. Pinot supports the following data types:
+
```

Comment:
I do see `Struct`, `Map`, `List` also in the code, but not sure of their current implementation status, so perhaps ok to skip in this PR.